src/Collapse.js

```diff
@@ -115,84 +115,82 @@ class Collapse extends React.Component {
 
 Collapse.propTypes = {
   /**
-   * Collapse the Component in or out.
+   * Whether the component is entered; triggers the enter or exit animation
```

Comment:
some vague description.
sorry but I did not get it. :sweat: 
may be add a couple simple one liner examples ?

--
though maybe it's just advanced English